syntax = "proto3";
package im;
import "im.proto"; 

service LogicService {
    rpc Login (LoginReq) returns (LoginRes);
    rpc SendMsg (MsgSendReq) returns (MsgSendRes);
}